components/Events.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Heading from "./Heading";
+import { UnderlineTypes } from "../utils/underlineType";
+import TextSection from "./TextSection";
+import Carousel from "./Carousel";
+import { EVENTS_IMAGES } from "../data/events";
+import { Navigation } from "swiper";
+import { SwiperSlide } from "swiper/react";
+import Image from "next/image";
+
+const EVENTS_CONTAINER = "flex flex-col h-84 bg-[#BAFBE4] pt-10 z-0";
+const CONTENT_CONTAINER = "flex flex-col pt-14 p-5 md:px-32";
+const CAROUSEL_CONTAINER = "text-white rounded-lg h-[40rem] shadow-2xl";
+const EVENTS_HEADER = "flex flex-row";
+const SPACE_BETWEEN_ELEMENTS = 50;
+
+const LONG_SQUIGGLY_LINE_SVG = "/svgs/events/long_events_squiggly_line.svg";
+
+const Events = () => {
+  const EventsCarousel = () => (
```

Comment:
Can we reuse the carousel we have for the alumni or the roles instead of creating a new one?